lambda-comprehend-language-sam-nodejs/template.yaml

```diff
@@ -0,0 +1,27 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: An AWS Lambda function that detects the Dominant Language in the provided input text using Amazon Comprehend.
+Resources:
+  DetectLanguageLambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: index.handler
+      Runtime: nodejs20.x
+      CodeUri: src/
+      MemorySize: 128
+      Timeout: 40
+      Architectures:
+        - arm64
+      Policies:
+        - Version: '2012-10-17'
+          Statement:
+          - Sid: LambdaDetectDominantLanguage
+            Effect: Allow
+            Action:
+            - comprehend:DetectDominantLanguage
+            Resource: '*'       
```

Comment:
Thanks for the pattern @abilashkumar 

Is this * resource required? Can we lock this down at all?

Reply:
@boyney123 This is required as the permission here is not tied to a resource, but in general to perform language detection on free text provided as an input. The "*" is required (it does not involve any customer resource or their saved configurations). A similar pattern published [earlier](https://github.com/aws-samples/serverless-patterns/blob/main/lambda-translator/template.yaml#L20).